conf/app.routes

```diff
@@ -1,3 +1,4 @@
 # microservice specific routes
 
 POST       /declarations            uk.gov.hmrc.cdsimportsdds.controllers.DeclarationController.create()
+GET        /declarations            uk.gov.hmrc.cdsimportsdds.controllers.DeclarationController.findAll()
```

Comment:
can we not name this one as `fetchDeclarations()` and the post one as `saveDeclarations()`, to not use mongo operation/action names?

Reply:
yep, will do